CMS/settings.py

```diff
@@ -87,6 +96,19 @@
 }
 
 
+DATABASES = {
+    'default': {
+        "ENGINE": 'django.db.backends.postgresql',
```

Comment:
I feel like we are still hardcoding the databse here. What we can do is to create a `.env.sample file` to store this environment variables

Reply:
Yes. And also include the Django Secret Key and the Allowed Host values in the .env file